src/org/loklak/harvester/strategy/KaizenQueries.java

```diff
@@ -26,7 +30,7 @@ public static KaizenQueries getDefaultKaizenQueries(int qLimit) {
             int queryLimit = qLimit;
 
             @Override
-            public boolean addQuery(String query) {
+            public boolean addQuery(String query, double score) {
```

Comment:
Where is score variable used in this function? I don't understand the use of the variable.